src/sync/deployment.test.ts

```diff
@@ -482,14 +382,14 @@ describe("sync/deployments", () => {
 				"deploymentSequenceNumber": 500226426,
 				"updateSequenceNumber": 500226426,
 				"displayName": "[TEST-123] test-commit-message",
-				"url": null,
+				"url": "random-success",
```

Comment:
Previous test didn't test the ff on scenario, how with ff on, this value will be extracted correctly.